spec/openhab/core/items/semantics_spec.rb

```diff
@@ -60,14 +62,41 @@
 
   describe "#points" do
     it "returns siblings of a point" do
-      expect(Patio_Light_Brightness.points).to eql [Patio_Light_Color]
+      expect(Patio_Light_Brightness.points).to eql [Patio_Light_Color, Patio_Light_Power, Patio_Light_ControlPower]
     end
 
     describe "returns points of an equipment" do
-      specify { expect(Patio_Light_Bulb.points).to match_array([Patio_Light_Brightness, Patio_Light_Color]) }
+      specify do
+        expect(Patio_Light_Bulb.points).to match_array([Patio_Light_Brightness,
+                                                        Patio_Light_Color,
+                                                        Patio_Light_Power,
+                                                        Patio_Light_ControlPower])
+      end
+
       specify { expect(Patio_Light_Bulb.points(Semantics::Light)).to eql [Patio_Light_Color] }
       specify { expect(Patio_Light_Bulb.points(Semantics::Level)).to eql [Patio_Light_Brightness] }
       specify { expect(Patio_Light_Bulb.points(Semantics::Level, Semantics::Control)).to eql [Patio_Light_Brightness] }
+      specify { expect(Patio_Light_Bulb.points([Semantics::Measurement, Semantics::Power])).to eql [Patio_Light_Power] }
+
+      specify do
+        expect(Patio_Light_Bulb.points(Semantics::Control, Semantics::Measurement))
+          .to match_array([Patio_Light_Brightness,
+                           Patio_Light_Color,
+                           Patio_Light_Power,
+                           Patio_Light_ControlPower])
+      end
+
+      specify do
+        expect(Patio_Light_Bulb.points(Semantics::Control, Semantics::Light, Semantics::Power))
+          .to match_array([Patio_Light_Color,
+                           Patio_Light_ControlPower])
+      end
```

Comment:
Will it also work for `points(Semantics::Control, Semantics::Measurement, Semantics::Power)` (two points, one common Property)

How about `points(Semantics::Control, Semantics::Measurement, Semantics::Power, Semantics::Light)` (two points, two properties)?

Reply:
Yes. Specs added.